docs/3.0.0/docs/canton/usermanual/pruning.rst

```diff
@@ -43,6 +43,69 @@ For the maximum duration to specify a reliable pruning window "end time", the le
 should should not be wildcards (`*`) as illustrated in the examples above. If the hour field is fixed, so should the
 fields for minute and second.
 
+Schedule Format
+---------------
+
+The format used to describe cron expression consists of 7 whitespace-separated fields:
+
+#. seconds
+#. minutes
+#. hours
+#. day-of-month
+#. month
+#. day-of-week
+#. year (optional)
+
+Each field has a set of valid values:
+
+* seconds and minutes must be a number between ``0`` and ``59``
+
+* hours must be a number between ``0`` and ``23``
+
+* day of the month must be a number between ``1`` and ``31``
+
+* month must be either
+
+  #. a number between ``0`` and ``11`` (with ``0`` being January), or
+
+  #. the first three letters of the month name (``JAN``, ``FEB``, ``MAR``, ...)
+
+* day of the week must be either
+
+  #. a number between ``1`` and ``7`` (with ``1`` being Sunday), or
+
+  #. the first three letters of the weekday name (``SUN``, ``MON``, ``TUE``, ...)
+
+* year must be a number between ``1900`` and ``2099``
+
+Note that while a day-of-month value might be valid by the definition above, it might not exist at all (like the
+31st of November). If you set a schedule to tick every month on the 31st, every month with less than 31 days will
+be skipped.
```

Comment:
```suggestion
Note that although a day-of-the-month value might be valid according to the above definition, it might not correspond to an actual date in certain months (such as the
31st of November). If you schedule pruning for the 31st of the month, every month with fewer than 31 days will
be skipped.
```